src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -453,12 +438,7 @@ ReportActionItemMessageEdit.propTypes = propTypes;
 ReportActionItemMessageEdit.defaultProps = defaultProps;
 ReportActionItemMessageEdit.displayName = 'ReportActionItemMessageEdit';
 
-export default compose(
-    withLocalize,
-    withReportActionsDrafts({
-        propName: 'drafts',
-    }),
-)(
+export default withLocalize(
```

Comment:
This is already existing code but as we're touching this component, let's use `useLocalize` hook.

Reply:
`withLocalizePropTypes` is no longer needed

Reply:
@situchan Updated